charts/super-agent/charts/super-agent-deployment/templates/preinstall-job-register-system-identity.yaml

```diff
@@ -79,45 +112,130 @@ spec:
               openssl rsa -in "$TEMPORAL_FOLDER/key" -pubout -out "$TEMPORAL_FOLDER/pub"
 
               echo Key generated, creating the identity...
-              for RETRY in 1 2 3; do
-                HTTP_CODE=$(echo '{ "query":
-                    "mutation {
-                      systemIdentityCreate(
-                        name: \"System Identity for Kubernetes cluster '{{ include "newrelic.common.cluster" . }}'\",
-                        organizationId: \"{{ include "newrelic-super-agent.auth.organizationId" . }}\",
-                        publicKey: \"'$(openssl enc -base64 -A -in "$TEMPORAL_FOLDER/pub")'\"
-                      ) {
-                        clientId,
-                        name
-                      }
-                    }"
-                  }' | tr -d $'\n' | \
-                  curl \
-                    -w "%{http_code}" \
+              if [ "${NEW_RELIC_AUTH_CLIENT_ID:-}" != "" ] && [ "${NEW_RELIC_AUTH_CLIENT_SECRET:-}" != "" ]; then
```

Comment:
I am not reviewing this snippet, I understand it is a copy/paste from onhost solution